extensions/PhabBugz/lib/WebService.pm

```diff
@@ -98,15 +101,53 @@ sub revision {
     };
 }
 
+sub check_user_permission_for_bug {
+    my ($self, $params) = @_;
+
+    my $user = Bugzilla->login(LOGIN_REQUIRED);
+
+    # Ensure PhabBugz is on
+    ThrowUserError('invalid_phabricator_request') unless Bugzilla->params->{phabricator_enabled};
+
+    # Validate that the requesting user's email matches phab-bot
+    ThrowUserError('unauthorized_user') unless $user->login eq PHAB_BMO_USER_EMAIL;
+
+    # Validate that a bug id and user id are provided
+    ThrowUserError('invalid_phabricator_request_params')
+      unless ($params->{bug_id} && $params->{user_id});
+
+    # Validate that the user and bug exist
+    my $user = Bugzilla::User->new($params->{user_id});
```

Comment:
this should be Bugzilla::user->check(..).  it will throw an invalid user error for you if it doesn't exist.